app/views/govuk_publishing_components/components/docs/organisation_logo.yml

```diff
@@ -69,6 +69,13 @@ examples:
         url: '/government/organisations/ministry-of-defence'
         brand: 'ministry-of-defence'
         crest: 'mod'
+  prime_ministers_office_10_downing_street:
```

Comment:
Should we remove the similar example of this from lines 51-57?

Reply:
@andysellick I've changed it to Cabinet Office as that page uses the executive office crest.

Reply:
Is that correct - I'm not sure what the strict definition of the executive office is?

Reply:
Good point, have just changed it to "Example" instead to be safe.